pkg/cache/cache.go

```diff
@@ -360,6 +360,37 @@ func (c *Cache) ForgetWorkload(w *kueue.QueuedWorkload) error {
 	return nil
 }
 
+// UpdateWorkloadStatus is a wrapper to update the condition of a workload
+// sets a new current condition and stores it on Conditions
+func (c *Cache) UpdateWorkloadStatus(ctx context.Context, wl *kueue.QueuedWorkload, wlct kueue.QueuedWorkloadConditionType, message, reason string) error {
```

Comment:
This function isn't one that updates/reads the cache, just put it in the file where it is used, queuedworkload_controller.go.

Reply:
moved